contrib/greetd/patches/config.patch

```diff
@@ -0,0 +1,17 @@
+--- a/config.toml	2023-08-29 22:25:37.253885597 +0200
++++ b/config.toml	2023-08-29 22:24:23.032078385 +0200
+@@ -1,7 +1,7 @@
+ [terminal]
+ # The VT to run the greeter on. Can be "next", "current" or a number
+ # designating the VT.
+-vt = 1
++vt = 7
```

Comment:
is there a reason not to use the default?

Reply:
if https://man.sr.ht/~kennylevinsen/greetd/#changing-target-vt is followed, i assume starting on 1 (or 1-6 rather) is already claimed by getty. i don't think it breaks, but '7' is kind of an unspecified default for spawned 'login' things?

that said, not sure why the default is left at 1 in that case

Reply:
in that case yeah (chimera runs gettys at 1-6 by default)